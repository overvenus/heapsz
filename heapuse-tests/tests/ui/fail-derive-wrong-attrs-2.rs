use heapuse_derive::Heap;
use heapuse_example::*;

#[derive(Heap)]
pub struct WrongAttr {
    #[heap(with)]
    pub a: primitive::StructPrimitives,
}

fn main() {}
