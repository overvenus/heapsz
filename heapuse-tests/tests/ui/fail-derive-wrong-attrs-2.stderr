error: heap attribute `with` miss mod path, it should be `with ="your::mod"`
 --> tests/ui/fail-derive-wrong-attrs-2.rs:6:12
  |
6 |     #[heap(with)]
  |            ^^^^
